### PR TITLE
Only apply automatic entity updates if option is set.

### PR DESCRIPTION
**Pull request by jhedstrom**
- Corresponding Drupal core issue: https://www.drupal.org/node/2542748

This can probably be merged prior to the core issue.

### CODE REVIEW

**Comment by weitzman** [comment]
Is there a reason one would not want entity updates? And if so, should the default be 'no entity updates'? That looks unintuitive. Is that how update.php works?

D8 only options get unhidden in a help_alter hook. See sql_drush_help_alter().

**Comment by jhedstrom** [comment]
The core critical is discussing not running them in update.php. However, the premise of that issue is  based on a [bug that was causing them to silently fail](https://www.drupal.org/node/2544954). I think drush should eventually follow core, and if core removes automatic updates from update.php, drush should add an option to run them.

File: commands/core/drupal/update.inc [hunk 1]
```diff
@@ -216,6 +220,26 @@ function drush_update_batch() {
   \Drupal::service('state')->set('system.maintenance_mode', FALSE);
 }
 
+/**
+ * Apply entity schema updates.
+ */
+function drush_update_entity_definitions(&$context) {
```
Comment (plach79) [inline commands/core/drupal/update.inc]: IMO it would be great to have a also a standalone command for this.

Comment (jhedstrom) [inline commands/core/drupal/update.inc]: That makes sense to me.

**Comment by jhedstrom** [comment]
I've added this as a separate command (`entity-updates`) as well.

**Comment by jhedstrom** [comment]
https://www.drupal.org/node/2542748 has been committed. Any last thoughts before I merge this?

**Comment by weitzman** [comment]
Nope - go for it.

**Comment by plach79** [comment]
RTBM on my end

File: commands/core/drupal/update.inc [hunk 2]
```diff
@@ -216,6 +220,26 @@ function drush_update_batch() {
...
+  try {
+    \Drupal::service('entity.definition_update_manager')->applyUpdates();
```
Comment (plach79) [inline commands/core/drupal/update.inc]: Minor thing: now we have `\Drupal::entityDefinitionUpdateManager()` :)

Comment (jhedstrom) [inline commands/core/drupal/update.inc]: Good to know! I've updated the PR to use that, and will merge after tests run.

**Comment by jhedstrom** [comment]
The failing test looks to be an unrelated issue with the libraries module. Committing this now.

**Pull request merged**